From aa000fd06865892b731e1bcc3b54634f9b242b22 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Wed, 31 Aug 2011 13:28:37 +0800
Subject: ENGR00155612-4 [mx6q]change the delay after clock frequence change
 to 1ms

100ms is too long delay, thus it impact other tasks scheduling.
for example, nfs reports timeout if two sd card is inserted because the
100ms delay occupies cpu too long.
1ms value is evaluated by IC engineer.

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 drivers/mmc/host/sdhci-esdhc.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-esdhc.h b/drivers/mmc/host/sdhci-esdhc.h
index c78ce0c..8ee1d62 100644
--- a/drivers/mmc/host/sdhci-esdhc.h
+++ b/drivers/mmc/host/sdhci-esdhc.h
@@ -87,7 +87,7 @@ static inline void esdhc_set_clock(struct sdhci_host *host, unsigned int clock)
 		| (div << ESDHC_DIVIDER_SHIFT)
 		| (pre_div << ESDHC_PREDIV_SHIFT));
 	sdhci_writel(host, temp, ESDHC_SYSTEM_CONTROL);
-	mdelay(100);
+	mdelay(1);
 
 	/* if there's board callback function
 	 * for pad setting change, that means
-- 
1.7.9.5

